### PR TITLE
LPS-88696

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java [hunk 1]
```diff
@@ -332,19 +349,17 @@ private UserWrapper _updateUser(
...
-				personUpdaterForm.getBirthdayYearOptional(),
-				birthdayDate.getYear()),
+				personUpdaterForm.getBirthdayYearOptional(), birthdayYear),
 			contact.getSmsSn(), contact.getFacebookSn(), contact.getJabberSn(),
 			contact.getSkypeSn(), contact.getTwitterSn(),
-			personUpdaterForm.getJobTitle(), user.getGroupIds(),
-			user.getOrganizationIds(), user.getRoleIds(), null,
-			user.getUserGroupIds(), new ServiceContext());
+			_getDefaultValue(
+				personUpdaterForm.getJobTitle(), user.getJobTitle()),
```
Comment (nhpatt) [inline modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java]: personUpdateForm should return an Optional and you won't need a new method

File: modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java [hunk 2]
```diff
@@ -321,6 +330,14 @@ private UserWrapper _updateUser(
 
 		Date birthdayDate = user.getBirthday();
 
+		Calendar calendar = Calendar.getInstance();
+
+		calendar.setTime(birthdayDate);
+
+		int birthdayDay = calendar.get(Calendar.DATE);
```
Comment (nhpatt) [inline modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java]: How is sorted in other resources? I don't know if you have to sort it alphabetically or by use.

Comment (dgarciasarai) [reply modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java]: https://github.com/liferay/liferay-portal/blob/79fd15fc19427c804c90143f37b96a3466be6c36/modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/form/PersonCreatorForm.java#L201

**Comment by dgarciasarai** [comment]
Changes done @nhpatt 👍

**Pull request merged**